packages/onboarding/src/index.ts

```diff
@@ -32,13 +32,3 @@ export * from './utils';
 export type { SelectItem } from './select-items';
 export type { SelectItemAlt } from './select-items-alt';
 export type { MShotsOptions } from './mshots-image';
-
-// Due to TypeScript complaining when building we need to export the type and the function separately.
-import {
-	hexToRgb,
-	LINK_IN_BIO_DOMAIN_FLOW,
-	LINK_IN_BIO_TLD_FLOW,
-	ECOMMERCE_FLOW,
-	WOOEXPRESS_FLOW,
-} from './utils';
-export { hexToRgb, LINK_IN_BIO_DOMAIN_FLOW, LINK_IN_BIO_TLD_FLOW, ECOMMERCE_FLOW, WOOEXPRESS_FLOW };
```

Comment:
I don't really know why I had to do this, but helped me to remove many warnings/errors that now does not appear any more...

Reply:
Notice the `LINK_IN_BIO_TLD_FLOW`, `hexToRgb`, etc.

```
# This file contains the result of Yarn building a package (wp-calypso@workspace:.)
# Script name: postinstall

packages/design-picker/src/components/style-variation-badges/utils.ts(1,10): error TS2305: Module '"@automattic/onboarding"' has no exported member 'hexToRgb'.
packages/domain-picker/src/utils/index.ts(4,2): error TS2724: '"@automattic/onboarding"' has no exported member named 'LINK_IN_BIO_TLD_FLOW'. Did you mean 'LINK_IN_BIO_FLOW'?
packages/domain-picker/src/utils/index.ts(5,2): error TS2305: Module '"@automattic/onboarding"' has no exported member 'ECOMMERCE_FLOW'.
packages/domain-picker/src/utils/index.ts(6,2): error TS2305: Module '"@automattic/onboarding"' has no exported member 'WOOEXPRESS_FLOW'.
client/odie/query/index.ts(2,19): error TS7016: Could not find a declaration file for module 'calypso/lib/wp'. '/Users/daniellopez/Repositories/wp-calypso/client/lib/wp/node.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/wp` if it exists or add a new declaration (.d.ts) file containing `declare module 'calypso/lib/wp';`
client/odie/message/index.tsx(5,25): error TS2307: Cannot find module 'calypso/assets/images/odie/wapuu-avatar.svg' or its corresponding type declarations.
client/odie/message/index.tsx(6,23): error TS7016: Could not find a declaration file for module 'calypso/components/async-load'. '/Users/daniellopez/Repositories/wp-calypso/client/components/async-load/index.jsx' implicitly has an 'any' type.
  Try `npm i --save-dev @types/calypso` if it exists or add a new declaration (.d.ts) file containing `declare module 'calypso/components/async-load';`
client/odie/wapuu-ribbon/index.tsx(4,22): error TS2307: Cannot find module 'calypso/assets/images/odie/wapuu-ear.svg' or its corresponding type declarations.
client/odie/wapuu-ribbon/index.tsx(5,19): error TS2307: Cannot find module 'calypso/assets/images/odie/wapuu-noear-notail.svg' or its corresponding type declarations.
client/odie/wapuu-ribbon/index.tsx(6,23): error TS2307: Cannot find module 'calypso/assets/images/odie/wapuu-tail.svg' or its corresponding type declarations.
client/odie/index.tsx(4,24): error TS7016: Could not find a declaration file for module 'calypso/components/back-button'. '/Users/daniellopez/Repositories/wp-calypso/client/components/back-button/index.jsx' implicitly has an 'any' type.
  Try `npm i --save-dev @types/calypso` if it exists or add a new declaration (.d.ts) file containing `declare module 'calypso/components/back-button';`
client/odie/index.tsx(5,32): error TS7016: Could not find a declaration file for module 'calypso/lib/analytics/track-component-view'. '/Users/daniellopez/Repositories/wp-calypso/client/lib/analytics/track-component-view/index.jsx' implicitly has an 'any' type.
  Try `npm i --save-dev @types/calypso` if it exists or add a new declaration (.d.ts) file containing `declare module 'calypso/lib/analytics/track-component-view';`
```